Specify that clock stands are allowed. (#890)
thewca/wca-regulations: wca-regulations.md

- F1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
- F2) The judge places the scrambled puzzle onto the mat in a standing position.

>     - F2a) A stand may be used to hold the puzzle in standing position.

Reviewer: I would prefer a more similar wording to `A2b1) For Square-1, the organization team may enforce placing a thin object in the puzzle to prevent accidental moves from being applied before the start of the attempt. If these objects are used, this must be announced before the round starts.`

```suggestion
    - F2a) The organization team may enforce placing a stand in the puzzle to prevent accidental falling before the start of the attempt. If these objects are used, this must be announced before the round starts.
```

Author: I don't really like the way A2b1 does this for now. But I think that it's reasonable to copy for now.

How's this?

```suggestion
    - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If these objects are used, this must be announced before the round starts.
```

Author: And since this was unclear enough that @hanwu85 asked about it:

```suggestion
    - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If these objects are used, this must be announced before the round starts.
        - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.
```

- F1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
- F2) The judge places the scrambled puzzle onto the mat in a standing position.

>     - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If these objects are used, this must be announced before the round starts.

Author: More consistent wording:

```suggestion
    - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If such stands are used, this must be announced before the round starts.
```

>         - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.

Reviewer: I’m not sure if this should be a “must”
